addons/web_editor/tests/test_tools.py

```diff
@@ -52,7 +64,22 @@ def test_get_video_url_data(self):
         #youtube
         self.assertEqual('youtube', tools.get_video_url_data(TestVideoUtils.urls['youtube'])['platform'])
         #vimeo
-        self.assertEqual('vimeo', tools.get_video_url_data(TestVideoUtils.urls['vimeo'])['platform'])
+        self.assertEqual(tools.get_video_url_data(TestVideoUtils.urls['vimeo']), {
```

Comment:
Could create a small loop on "vimeo", "vimeo_player" and assert inside, as they seem to have the same values?